addons/project/views/project_project_views.xml

```diff
@@ -550,10 +550,10 @@
                     color="color">
                     <field name="partner_id" attrs="{'invisible': [('partner_id', '=', False)]}"/>
                     <field name="user_id" widget="many2one_avatar_user" attrs="{'invisible': [('user_id', '=', False)]}"/>
-                    <field name="is_favorite" widget="boolean_favorite" nolabel="1" string="Favorite"/>
-                    <field name="stage_id" groups="project.group_project_stages"/>
+                    <field name="is_favorite" invisible="1"/>
```

Comment:
Is it really needed to add it in the view if the field is always invisible? 🤔

Reply:
Oops!! thanks for noticing it but as we are not moved favorite from popover to right next to title/name, we should leave it as it was before.